tests/itests/src/test/java/org/codice/ddf/admin/query/AdminSecurityIT.java

```diff
@@ -0,0 +1,286 @@
+
+    public static ServiceReader serviceReader;
+
+    @Override
+    public List<Feature> features() {
+        // TODO: tbatie - 8/19/17 - Fix all these features once they've been fixed in DDF. We should only have to call our admin-console-app feature
+        return Arrays.asList(PlatformAppFeatureFile.featureFile(),
+                SecurityAppFeatureFile.featureFile(),
+                ComponentTestFeatureFile.thirdPartyFeature()
+                        .bootFeature(true),
+                ComponentTestFeatureFile.commonTestDependenciesFeature()
+                        .bootFeature(true),
+                AdminQueryAppFeatureFile.adminCoreFeature()
+                        .bootFeature(true),
+                //  add so pax exam has access to data type api before being started
+
+                //Added a boot feature because the tests need the ServiceManager running before we can startFeatures
+                ComponentTestFeatureFile.securityAll()
+                        .bootFeature(true),
+                ComponentTestFeatureFile.configuratorFeature(),
+                ComponentTestFeatureFile.configSecurityPolicy(),
+                ComponentTestFeatureFile.ldapClaimsHandlerFeature(),
+                ComponentTestFeatureFile.ldapLoginFeature(),
+
+                AdminQueryAppFeatureFile.adminSecurityFeature(),
+                AdminQueryAppFeatureFile.adminGraphQlFeature());
+    }
+
+    @BeforeExam
+    public void beforeTest() throws Exception {
+        super.beforeExam();
+        securityPolicyConfigurator.configureRestForBasic();
+
+        WCPM_REQUEST_HELPER.waitForWcpmInSchema();
+        LDAP_REQUEST_HELPER.waitForLdapInSchema();
+        serviceReader = serviceManager.getService(ServiceReader.class);
+    }
+
+    @Test
+    public void getAuthTypes() throws Exception {
+        assertThat(WCPM_REQUEST_HELPER.getAuthType()
+                .isEmpty(), is(false));
+    }
+
+    @Test
+    public void getRealms() throws IOException {
+        assertThat(WCPM_REQUEST_HELPER.getRealms()
+                .isEmpty(), is(false));
+    }
+
+    @Test
+    public void saveWhiteListed() throws IOException {
+        try {
+            List<String> expectedWhiteListValues = new ArrayList<>();
+            expectedWhiteListValues.addAll(WCPM_REQUEST_HELPER.getWhiteListContexts());
+            expectedWhiteListValues.add(TEST_CONTEXT_PATH);
+
+            WCPM_REQUEST_HELPER.saveWhiteListContexts(expectedWhiteListValues);
+            WCPM_REQUEST_HELPER.waitForWhiteList(expectedWhiteListValues);
+        } finally {
+            WCPM_REQUEST_HELPER.resetWhiteList();
+        }
+    }
+
+    /**
+     * For the user's convenience, identical policy bins are collapsed into a single policy bin. This
+     * test confirms two policies bins can be persisted and retrieved as a single collapsed policy
+     *
+     * @throws IOException
+     */
+    @Test
+    public void savePolicies() throws IOException {
+
+        try {
```

Comment:
Yes, they can and should be. Unfortunately I wrote these tests at the same time I was fixing the bins collapsing to make my life easier. I'll leave a todo to move the collapsing bins test to the unit test level.